tests/unitary/src/compute/test_host_buffer.cpp

```diff
@@ -54,6 +54,49 @@ class mock_host_buffer final
 
 
 
+TEST_CASE( "get_device_accessible_alias should return null when null is provided", "[host_buffer]" )
+{
+    std::shared_ptr<host_buffer> buffer;
+    REQUIRE( get_device_accessible_alias(buffer) == nullptr );
+    std::shared_ptr<const host_buffer> const_buffer;
+    REQUIRE( get_device_accessible_alias(const_buffer) == nullptr );
+}
+
+TEST_CASE( "get_device_accessible_alias should return null when buffer is not aliasble", "[host_buffer]" )
```

Comment:
aliasable

Reply:
Done in [c1d7163](https://github.com/gigabit-clowns/xmipp4-core/pull/119/commits/c1d7163b22e90ca05754cc2208d6a54cf6a8a251)